src/jit/instr.h

```diff
@@ -296,7 +296,9 @@ enum InstructionSet
 {
     InstructionSet_ILLEGAL = 0,
 #ifdef _TARGET_XARCH_
-    InstructionSet_Base,
+    InstructionSet_Vector64,
```

Comment:
nit: I think these can be removed here now, and same for `InstructionSet_Vector256` under `TARGET_ARM64`